dist/enhanced-shutter-card.js

```diff
@@ -1226,17 +1228,28 @@ class EnhancedShutter extends LitElement
     //console_log('Shutter Resize detected',this[ESC_CLASS_SELECTOR]?.getBoundingClientRect());
     //console_log('Shutter Update ready');
   }
-  firstUpdated(changedProperties){
-    //console_log('Shutter firstUpdated');
-    this[ESC_CLASS_SELECTOR]        = findElement(this,'.'+ESC_CLASS_SELECTOR);
-    this[ESC_CLASS_SELECTOR_SLIDE]  = findElement(this,'.'+ESC_CLASS_SELECTOR_SLIDE);
-    this[ESC_CLASS_SELECTOR_PICKER] = findElement(this,'.'+ESC_CLASS_SELECTOR_PICKER);
-    // resize observer here .... observer needs  this[ESC_CLASS_SELECTOR] to be set
-    //displayNodePathToTopIncludingShadowAndClass(this[ESC_CLASS_SELECTOR]);
-    this.startResizeObserver();
-    //console_log('Shutter firstUpdated ready');
+  firstUpdated(changedProperties) {
+    super.firstUpdated(changedProperties);
+
+    this[ESC_CLASS_SELECTOR]        = findElement(this, `.${ESC_CLASS_SELECTOR}`);
+    this[ESC_CLASS_SELECTOR_SLIDE]  = findElement(this, `.${ESC_CLASS_SELECTOR_SLIDE}`);
+    // NOTE: drop the old lit‐decorated @touchstart from the template
+    //      so we can re‑attach it manually below
+    const picker = findElement(this, `.${ESC_CLASS_SELECTOR_PICKER}`);
+    if (picker) {
+      // detach any lit‐added touchstart so we don’t double‐fire
+      picker.removeEventListener('touchstart', this.mouseDown);
```

Comment:
Is this still needed?? 
You removed all the @ commands, (@touchstart and the other 2).
 So a remove is not needed anymore, I think.